### PR TITLE
SQLonlinux_mssconf_docupdate

### PR DESCRIPTION
**Pull request by aravindmahadevan-ms**
mssqlconf doc update - ldaphostcannon

### CODE REVIEW

**Comment by PRMerger19** [comment]
@aravindmahadevan-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Court72** [comment]
@VanMSFT 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by VanMSFT** [comment]
@amvin87 - Hey Amit, can you please help confirm the additions proposed here? Thanks!

File: docs/linux/sql-server-linux-configure-mssql-conf.md [hunk 1]
```diff
@@ -691,6 +691,7 @@ The following options are additional network settings configurable using `mssql-
 | network.kerberoscredupdatefrequency | Time in seconds between checks for kerberos credentials that need to be updated. Value is an integer.|
 | network.privilegedadaccount | Privileged AD user to use for AD authentication. Value is `<username>`. For more information, see [Tutorial: Use Active Directory authentication with SQL Server on Linux](sql-server-linux-active-directory-authentication.md#spn)|
 | uncmapping | Maps UNC path to a local path. For example, `sudo /opt/mssql/bin/mssql-conf set uncmapping //servername/sharename /tmp/folder`. |
+| ldaphostcannon | Set whether OpenLDAP should canonicalize hostnames during the bind step , Values can be `true` or `false`.|
```
Comment (rwestMSFT) [inline docs/linux/sql-server-linux-configure-mssql-conf.md:694]:
```suggestion
| ldaphostcanon | Set whether OpenLDAP should canonicalize hostnames during the bind step. Values can be `true` or `false`.| 
```

**Comment by rwestMSFT** [comment]
#assign:amvin87

**Review by amvin87 — APPROVED** [review]

**Comment by amvin87** [comment]
#sign-off

**Comment by VanMSFT** [comment]
Since this has merge conflicts, @rwestMSFT is fixing this in https://github.com/MicrosoftDocs/sql-docs-pr/pull/24182. Thanks @aravindmahadevan-ms!

**Pull request closed without merging**